Project.toml

```
@@ -4,6 +4,7 @@ authors = ["Hendrik Ranocha <mail@ranocha.de> and contributors"]
 version = "2.19.2"
```

```suggestion
version = "2.20.0"
```
Let's bump the version of this package when including this new feature.